gtsam/geometry/Pose3.cpp

```diff
@@ -243,29 +243,24 @@ namespace pose3 {
 class ExpmapFunctor : public so3::DexpFunctor {
  private:
   static constexpr double one_twenty_fourth = 1.0 / 24.0;
-  static constexpr double one_one_hundred_twentieth = 1.0 / 120.0;
+
+  // Constant used in computeQ
+  double F;  // (B - 0.5) / theta2 or -1/24 for theta->0
 
  public:
   ExpmapFunctor(const Vector3& omega, bool nearZeroApprox = false)
-      : so3::DexpFunctor(omega, nearZeroApprox) {}
+      : so3::DexpFunctor(omega, nearZeroApprox) {
+        F = nearZero ? - one_twenty_fourth : (B - 0.5) / theta2;
+      }
 
   // Compute the bottom-left 3x3 block of the SE(3) Expmap derivative
   // TODO(Frank): t = applyLeftJacobian(v), it would be nice to understand
   // how to compute mess below from its derivatives in w and v.
```

Comment:
Still todo?

Reply:
Yes, because I cannot just use the applyLeftJacobian Jacobians. Which I think should be able to. Note Pose3 constructor Jacobian is 6x3 in R and 6x3 in t, with latter $[Z_3x3, R.transpose()]$, so my reading was Q = R.transpose()*H1, where H1 is apply jacobian for w.

Reply:
I think it's likely related to the A,B,C etc not a constant but a function of w. So you need to cache the raw double cross derivatives not the result of `applyLeftJacobian`.

Reply:
Yeah, but crossB and doubleCrossC explicitly account for that, e.g., D is related to derivative of B, and E is related to derivative of C. And they are (fairly) extensively unit-tested, and so is applyLeftJacobian.

Reply:
Note:
<img width="316" alt="image" src="https://github.com/user-attachments/assets/1571a8fe-94a7-4c32-9bbb-76c5bf8797af" />
And C is the only second order "constant". In the derivative of applyL, we have:
- $C$ * doubleCrossJacobian = $C*(w v I + v w^T - 2 w v^T )$
and in Q we have:
- $C * (W * V + V * W - W V W)$

It's weird that there even is a W V W here, as that is third-order.

Reply:
I solved it! The applyLeftJacobian yields two jacobians, $H_1$ and $H_2$, with H2 being leftJacobian $J_L$. But the corresponding block is the right Jacobian $J_R$. Hence, the action X of R^T in the mapping must be
$J_R = X J_L$
hence
$X = J_R J_L^{-1}$
I verified that $Q = X H_1$, will make a PR to that effect. Massively simpler.